### DISCUSSION TITLE
告警如何获取 cpu 数量

### DISCUSSION THREAD

**Discussion by daseinem** [zdz/ServerStatus-Rust#118] [Q&A] [answered]
我想实现当节点负载接近或大于 cpu 核心数时发送告警，但是似乎 [payload.rs ](https://github.com/zdz/ServerStatus-Rust/blob/master/server/src/payload.rs) 并没有这一项。

**Comment by zdz** [comment, marked as answer]
可以取 `sys_info.cpu_num `，另外可用的两个结构 `ip_info`, `sys_info `

https://github.com/zdz/ServerStatus-Rust/blob/20979d7809887e3315a8852cc2245066da1b0dac/common/proto/server_status.proto#L32

**Reply by daseinem** [reply]
明白了，非常感谢！
